lib/ansible/modules/system/authorized_key.py

```diff
@@ -233,16 +233,16 @@ def _item_generator(self):
             yield key, self[key][indexes[key]]
 
     def iteritems(self):
-        return self._item_generator()
+        raise NotImplementedError("Do not use this as it's not available on py3")
```

Comment:
Since the silly thing inherits from dict, if I just remove the iteritems method it will fall back to dict's iteritems.  That's not desirable for two reasons:
* dict's iteritems won't keep the keys ordered.
* We really don't want code using using iteritems since python3 doesn't have it.

(same rationale for replacing itervalues instead of removing it).